eel/tests/receiving_payments_test.rs

```diff
@@ -167,13 +178,20 @@ mod receiving_payments_test {
         paying_node: NodeInstance,
         payment_amount: u64,
         lsp_fee: u64,
+        lsp_topup: u64,
     ) {
-        let initial_balance = node.get_node_info().channels_info.local_balance_msat;
+        let channels_info = node.get_node_info().channels_info;
+        let initial_balance = channels_info.local_balance_msat;
+        let initial_channel_capacity = channels_info.total_channel_capacities_msat;
 
         let invoice = issue_invoice(&node, payment_amount);
 
         nigiri::pay_invoice(paying_node, &invoice).unwrap();
 
+        let channels_info = node.get_node_info().channels_info;
+        let new_channel_capacity =
+            channels_info.total_channel_capacities_msat - initial_channel_capacity;
+        assert_eq!(new_channel_capacity, lsp_topup + payment_amount);
```

Comment:
Would be cool if most of this new logic got extracted into a `assert_increased_capacity()` function :)